client/components/BotRunTestStatusList/BotRunTestStatusList.module.css

```diff
@@ -0,0 +1,10 @@
+.bot-run-test-container {
+  font-size: 0.875rem !important;
+  padding: 0.5rem 0;
+  margin: 0.5rem 0;
```

Comment:
It could be useful to have gap variables as well. A future improvement consideration

Reply:
Certainly! I've started defining them in https://github.com/w3c/aria-at-app/pull/1338/commits/f9dff83a2a6b5600bd444e97c4b27901ed3b16ab but it feels like an even bigger effort to go through and change those now, though I wish I considered it at the time 😅. So I think we can certainly have this as a future improvement